src/Compilers/CSharp/Portable/Symbols/NamespaceOrTypeSymbol.cs

```diff
@@ -153,19 +153,27 @@ internal virtual ImmutableArray<NamedTypeSymbol> GetTypeMembersUnordered()
         /// <returns>An ImmutableArray containing all the types that are members of this symbol with the given name.
         /// If this symbol has no type members with this name,
         /// returns an empty ImmutableArray. Never returns null.</returns>
-        public abstract ImmutableArray<NamedTypeSymbol> GetTypeMembers(string name);
+        public ImmutableArray<NamedTypeSymbol> GetTypeMembers(string name)
+            => GetTypeMembers(name.AsMemory());
```

Comment:
the original abstract methods become non-abstract, but bridge to the new ROM versions.  This keeps binary compar with all consumers in teh compiler layer that still operate using strings, while providing the new ROM versions for the codepaths that can take advantage of them.